Dockerfile

```diff
@@ -13,6 +13,13 @@ RUN pip3 install pandas==1.1.1
 RUN pip3 install jupyter==1.0.0 && pip3 install jupyterlab==2.1.0
 # https://stackoverflow.com/questions/49024624/how-to-dockerize-jupyter-lab
 
+# Lingvo ASR dependencies
+# supported versions: (lingvo==0.6.4 with tensorflow==2.1.0)
```

Comment:
I have noticed that `pip install lingvo==0.6.4` is installing `tensorflow-gpu` as a dependency even if `tensorflow` is installed. Do you know if there is a way to install TensorFlow only once if we install ART with the new option `lingvo_asr`?

Reply:
Yes, this is a typo in the comment :+1: